<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAuthorization</class>
 <widget class="QDialog" name="CAuthorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>268</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BuksanSpy</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Загрузки/icon.png</normaloff>../../../Загрузки/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="LoginAll_Layout">
     <item row="1" column="0">
      <layout class="QFormLayout" name="Login_Layout">
       <item row="0" column="0">
        <widget class="QLabel" name="host_Label">
         <property name="text">
          <string>Host:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Host_Edit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Database:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="DataBase_Edit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Port_Label">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="Port_Edit">
         <property name="text">
          <string>5432</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Login_Label">
         <property name="text">
          <string>Login:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Login_Edit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="Password_Label">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="Password_Edit">
         <property name="locale">
          <locale language="Russian" country="Russia"/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="TryLogin_Button">
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="SaveLoginStrings_checkBox">
         <property name="text">
          <string>Save Session (for 3 days)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="ShowPassword_checkBox">
         <property name="text">
          <string>Show Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
